2024-06-26 01:43:34,221 ERROR: Exception on /explore [GET] [in C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\app.py:828]
Traceback (most recent call last):
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\app\routes.py", line 192, in explore
    return render_template("explore.html", title='Explore', posts=posts.items,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\jinja2\loaders.py", line 125, in load
    source, filename, uptodate = self.get_source(environment, name)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: explore.html
2024-06-26 01:43:55,123 INFO: Microblog startup [in C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\app\__init__.py:45]
2024-06-26 01:43:57,157 ERROR: Exception on /explore [GET] [in C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\app.py:828]
Traceback (most recent call last):
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\app\routes.py", line 192, in explore
    return render_template("explore.html", title='Explore', posts=posts.items,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\templating.py", line 149, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\jinja2\loaders.py", line 125, in load
    source, filename, uptodate = self.get_source(environment, name)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\06atm\OneDrive\Masaüstü\B.P. 4.dönem\OOP II\microblog\venv\Lib\site-packages\flask\templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: explore.html
